src/Components/Patient/ManagePatients.tsx

```diff
@@ -295,12 +291,10 @@ export const PatientManager = (props: any) => {
 
   const fetchFacilityName = useCallback(
     async (status: statusType) => {
-      setIsLoading(true);
```

Comment:
Why do we have these changes?

Reply:
I was initially working on `issue-1404` branch and there is a commit by Sandra so I suppose these are those changes